next/src/field/type.ts

```diff
@@ -7,8 +7,9 @@ export interface Field {
   label?: string
   description?: string
   fields?: Field[]
-  type: string
-  inputType: string
+  // @deprecated in favor of inputType,
+  type: FieldType
```

Comment:
suggestion: Could we not return `type` by default? It's deprecated, we want it out of v1.

As internally we need it ourselves, we can pass a config key like `deprecatedFieldAttrType: true` to output it. WDYT?

Reply:
I'm happy to remove it tbh. If we assume this is a deprecation to be done now, I'd not add that config, I'd remove it completely.

Reply:
Still, I'd prefer to remove this on a future PR. It's safer to keep it, for the time being.